2.wave3_data/4.process_features/extract_sc_wave3.ipynb

```diff
@@ -0,0 +1,1702 @@
+       "2                                0.241474   \n",
+       "3                                0.152092   \n",
+       "4                                0.393185   \n",
+       "\n",
+       "  Nuclei_Texture_Variance_CorrPM_3_03_256  \\\n",
+       "0                                0.416597   \n",
+       "1                                 0.10911   \n",
+       "2                                0.238959   \n",
+       "3                                0.133185   \n",
+       "4                                0.388422   \n",
+       "\n",
+       "  Nuclei_Texture_Variance_CorrRNA_3_00_256  \\\n",
+       "0                                 0.408288   \n",
+       "1                                 0.367092   \n",
+       "2                                 0.276973   \n",
+       "3                                 0.372212   \n",
+       "4                                 0.338986   \n",
+       "\n",
+       "  Nuclei_Texture_Variance_CorrRNA_3_01_256  \\\n",
+       "0                                 0.381595   \n",
+       "1                                 0.362007   \n",
+       "2                                 0.276883   \n",
+       "3                                 0.402119   \n",
+       "4                                 0.328554   \n",
+       "\n",
+       "  Nuclei_Texture_Variance_CorrRNA_3_02_256  \\\n",
+       "0                                 0.411175   \n",
+       "1                                 0.354774   \n",
+       "2                                 0.281929   \n",
+       "3                                 0.393876   \n",
+       "4                                 0.321786   \n",
+       "\n",
+       "  Nuclei_Texture_Variance_CorrRNA_3_03_256  \n",
+       "0                                 0.452748  \n",
+       "1                                 0.329226  \n",
+       "2                                 0.272285  \n",
+       "3                                 0.376437  \n",
+       "4                                 0.319527  \n",
+       "\n",
+       "[5 rows x 2473 columns]"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Merge single cells across compartments\n",
+    "anno_kwargs = {\"join_on\": [\"Metadata_well\", \"Image_Metadata_Well\"]}\n",
+    "\n",
+    "sc_df = sc.merge_single_cells(\n",
+    "    platemap=platemap_df,\n",
+    "    **anno_kwargs,\n",
+    ")\n",
+    "\n",
+    "# Save level 2 data as a csv\n",
+    "output(sc_df, sc_output_file)\n",
+    "\n",
+    "print(sc_df.shape)\n",
+    "sc_df.head()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "52dc3130",
+   "metadata": {},
+   "source": [
+    "## Normalize data by DMSO 0.1% treatment"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "93e607fa",
+   "metadata": {},
+   "source": [
+    "Have to set this due to kernel breaking"
```

Comment:
Is this still necessary?

Reply:
Nope! I just removed it. The numbers for the `In[]` will be off by one but it would take over an hour to run it all again so it will just have to stay like that.